Kickstarter-iOS/Views/Cells/CommentRemovedCell.swift

```diff
@@ -0,0 +1,130 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class CommentRemovedCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var rootStackView = {
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var infoImageView = {
+    UIImageView(frame: .zero)
+      |> \.image .~ Library.image(named: "info")
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var commentLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var rowStackView: UIStackView = {
+    UIStackView(frame: .zero)
+  }()
+
+  private lazy var commentCellHeaderStackView: CommentCellHeaderStackView = {
+    CommentCellHeaderStackView(frame: .zero)
+  }()
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    self.setupConstraints()
+    self.bindStyles()
+    self.configureViews()
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> baseTableViewCellStyle()
+
+    _ = self.rootStackView
+      |> commentCellRootStackViewStyle
+
+    _ = self.rowStackView
+      |> rowStackViewStyle
+
+    _ = self.commentLabel
+      |> \.attributedText .~ attributedTextCommentRemoved()
+      |> \.lineBreakMode .~ .byWordWrapping
+      |> \.numberOfLines .~ 0
+      |> \.adjustsFontForContentSizeCategory .~ true
+  }
+
+  // MARK: - Configuration
+
+  internal func configureWith(value: DemoComment) {
+    self.commentCellHeaderStackView.configureWith(comment: value)
+  }
+
+  private func configureViews() {
+    self.rowStackView.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
+    _ = ([self.commentCellHeaderStackView, self.rowStackView], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.infoImageView, self.commentLabel], self.rowStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+  }
+
+  private func setupConstraints() {
+    _ = (self.rootStackView, self.contentView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToMarginsInParent()
+
+    NSLayoutConstraint.activate([
+      self.infoImageView.widthAnchor.constraint(equalToConstant: Styles.grid(4)),
```

Comment:
Does the image not have an intrinsic size?

Reply:
So because of how the design is on Figma, the width and height would need to be fixed, so the image can float at the top. Unless there's a better way to approach this.

Reply:
Instead of width constraints here can we use:

```swift
self.infoImageView.setContentHuggingPriority(.required, for: .horizontal)
```